src/ui/units/wizard/components/Dialogs/DialogPlaceholder/DialogPlaceholder.tsx

```diff
@@ -28,7 +28,8 @@ import {
     isNumberField,
     isPercentVisualization,
 } from 'shared';
-import {Utils, withHiddenUnmount} from 'ui';
+import {Utils} from 'ui';
```

Comment:
One more issue about imports( 
Can you use `ui/utils` or `ui/utils/utils`, please?